packages/library-legacy/scripts/typegen.sh

```diff
@@ -13,13 +13,10 @@ sed -i.bak '$s/export {.*};/}/' lib/index.d.ts
 sed -i.bak 's/declare/export/g' lib/index.d.ts
 
 # Prepend declare module line
-sed -i.bak '2s;^;declare module "@solana/web3.js" {\n;' lib/index.d.ts
+sed -i.bak '2s;^;declare module '\''@solana/web3.js'\'' {\n;' lib/index.d.ts
 
 # Remove backup file from `sed` above
 rm lib/index.d.ts.bak
 
 # Run prettier
 pnpm prettier --write lib/index.d.ts
-
-# Check result
-pnpm tsc --types ./test/__shadow-jest-types.d.ts lib/index.d.ts
```

Comment:
This never made any sense, and indeed no longer works in TS 5.5. `tsc` isn't designed to typecheck TypeScript definition files.